components/packaging/common/load-telemetry-plugin

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env sh
+
+BASE=/var/lib/docker
```

Comment:
Yep, but we can still rely on that `directory` for the plugin installation. I couldn't find a better way to do it.